crates/compiler/mono/src/layout.rs

```diff
@@ -2846,11 +2846,11 @@ impl<'a> Layout<'a> {
     }
 
     pub fn default_integer() -> InLayout<'a> {
-        Layout::I64
+        Layout::U128
```

Comment:
We want this to be I128 because otherwise, we won't get signed arithmetic with the default number type.

Reply:
Thank you ! I don't know why I did that ...